repose-aggregator/core/core-lib/src/main/java/org/openrepose/powerfilter/PowerFilter.java

```diff
@@ -374,6 +378,10 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
                 }
                 requestFilterChain.startFilterChain(mutableHttpRequest, mutableHttpResponse);
             }
+        } catch (IllegalArgumentException iae) {
```

Comment:
This is the only thing that makes me a little nervous. This is a fairly common exception, and I'd hate to log an invalid method when that isn't the cause.

One alternative would be to break the check into its own try block. That would let us fail earlier (not that we spend much time processing prior to this) and catch the exception on a very isolated portion of the code.

I'll defer to @adrianjgeorge